### PR TITLE
Change | Convert Netfx to SDK style

### PR DESCRIPTION
**Pull request by JRahnama**
This pull request converts the current existing .NET Framework project to SDK style. 
Merging the two projects could be the next steps, which require more time and consideration

### CODE REVIEW

**Comment by ErikEJ** [comment]
Is line 83-93 really still needed?

**Comment by JRahnama** [comment]
> Is line 83-93 really still needed?

Initially I took them out, but got an error complaining about System.Transaction and I had to put them back. I did not check with other ones 😄

**Comment by ErikEJ** [comment]
Fine to leave them in for now

**Comment by benrr101** [comment]
I'd say if it builds, it looks good to me 👍

**Comment by David-Engel** [comment]
> it doesn't look like I can review

🤔 You should definitely be able to review...

File: src/Microsoft.Data.SqlClient/tests/FunctionalTests/Microsoft.Data.SqlClient.Tests.csproj [hunk 1]
```diff
@@ -122,7 +122,12 @@
     <PackageReference Condition="$(ReferenceType.Contains('Package'))" Include="Microsoft.Data.SqlClient" Version="$(TestMicrosoftDataSqlClientVersion)" />
   </ItemGroup>
   <ItemGroup>
-    <None Condition="'$(TargetGroup)'=='netfx' AND $(ReferenceType)=='Project'" Include="$(BinFolder)$(Configuration).AnyCPU\Microsoft.Data.SqlClient\netfx\**\*SNI*.dll" CopyToOutputDirectory="PreserveNewest" />
+   <ContentWithTargetPath Condition="'$(TargetGroup)'=='netfx' AND $(ReferenceType)=='Project'" Include="$(BinFolder)$(Configuration).AnyCPU\Microsoft.Data.SqlClient\netfx\**\*SNI*.dll">
+    <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    <TargetPath>%(Filename)%(Extension)</TargetPath>
```
Comment (arellegue) [inline src/Microsoft.Data.SqlClient/tests/FunctionalTests/Microsoft.Data.SqlClient.Tests.csproj:127]: I am just wondering if a dot is required between the %(Filename) and %(Extension).

File: src/Microsoft.Data.SqlClient/tests/ManualTests/Microsoft.Data.SqlClient.ManualTesting.Tests.csproj [hunk 2]
```diff
@@ -347,7 +347,10 @@
     <PackageReference Condition="'$(TargetGroup)'!='netfx'" Include="System.ServiceProcess.ServiceController" Version="$(SystemServiceProcessServiceControllerVersion)" />
   </ItemGroup>
   <ItemGroup>
-    <None Condition="'$(TargetGroup)'=='netfx' AND $(ReferenceType)=='Project'" Include="$(BinFolder)$(Configuration).AnyCPU\Microsoft.Data.SqlClient\netfx\**\*SNI*.dll" CopyToOutputDirectory="PreserveNewest" />
+   <ContentWithTargetPath Condition="'$(TargetGroup)'=='netfx' AND $(ReferenceType)=='Project'" Include="$(BinFolder)$(Configuration).AnyCPU\Microsoft.Data.SqlClient\netfx\**\*SNI*.dll">
+    <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    <TargetPath>%(Filename)%(Extension)</TargetPath>
```
Comment (arellegue) [inline src/Microsoft.Data.SqlClient/tests/ManualTests/Microsoft.Data.SqlClient.ManualTesting.Tests.csproj:352]: I am just wondering if a dot is required between the %(Filename) and %(Extension).

**Review by arellegue — APPROVED** [review]
Looks good to me.

**Review by benrr101 — APPROVED** [review]

**Comment by benrr101** [comment]
> > it doesn't look like I can review
> 
> 🤔 You should definitely be able to review...

It's been a while since I've reviewed a PR on github ... didn't realize I had to go to the changed files tab to submit the review. We're all good now 👍

**Review by David-Engel — APPROVED** [review]

**Pull request merged**